readme.md

```diff
@@ -118,47 +118,47 @@ Once installation is working, try creating and requesting resources. Note that t
 
-* **Your Answer:**
+* **Your Answer:** The information we get when we use `.updateOne()` is meta data about the update. The difference when we use `.findOneAndUpdate()` method is that it returns the old version of the document that we just updated instead of meta data.
 
 * **Question:** This new method will return the _old document_ instead of the new one. What option can you add to your statement to return the new one?
 
-* **Your Answer:**
+* **Your Answer:** We need to pass in a third argument into `.findOneAndUpdate()` with the object `{new:true}`. This will return the new updated document instead of the old document that we updated.
 
 * **Question:** Another field was updated when you ran this command. Which one was it?
 
-* **Your Answer:**
+* **Your Answer:** The field/property that was also updated when we made the PUT request is the `updated_at` property.
 
 * **Question:** Take a look at the terminal window running your server. You are likely getting the following deprecation warning:
   ```
   DeprecationWarning: Mongoose: `findOneAndUpdate()` and `findOneAndDelete()` without the `useFindAndModify` option set to false are deprecated. See: https://mongoosejs.com/docs/deprecations.html#-findandmodify-
   ```
   Take a look [at this page](https://mongoosejs.com/docs/deprecations#findandmodify) to see how to fix it. Describe the changes that took place.
 
-* **Your Answer:**
+* **Your Answer:** We need to add the option `useFindAndModify:false` in our mongoose connection to prevent this DeprecationWarning from happening.
 
 ---
 
 - [ ] Update the API so that you can successfully delete a record. Use the [Model.findOneAndDelete()](https://mongoosejs.com/docs/api.html#model_Model.findOneAndDelete) method.
 
 ---
 
-- [ ] A lot of information is being returned on each request. Some of this information we may want to keep private. Using the [.select()](https://mongoosejs.com/docs/api.html#query_Query-select) method, return only the "_id", "title", "start_year", and "season_count" on all the above requests.
+- [ ] A lot of information is being returned on each request. Some of this information we may want to keep private. Using the [.select()](https://mongoosejs.com/docs/api.html#query_Query-select) method, return only the `_id`, `title`, `start_year`, and `season_count` on all the above requests.
 
 * **Question:** At least one of these will throw an error. Which one? How can you get around this issue?
 
-* **Your Answer:** 
+* **Your Answer:** None of the 4 properties throw an error. They are all returned. We may want to exclude some of the properties from being returned by adding the `-_id` inside the select to prevent the id field from being shown in our response.
```

Comment:
You should encounter an error when you run `.create().select()`. You will have to re-request the information or use the `.save()` method to return the updated record.